addons/crm/static/src/js/crm_kanban.js

```diff
@@ -4,12 +4,156 @@
+            this._super.apply(this, arguments);
+            if (this.showRecurringRevenue) {
+                this.prevTotalRecurringRevenue = this.totalRecurringRevenue;
+                this.totalRecurringRevenue = this.columnState.aggregateValues[this.recurringRevenueSumField] || 0;
+            }
+        },
+
+        //--------------------------------------------------------------------------
+        // Private
+        //--------------------------------------------------------------------------
+
+        /**
+         * @private
+         * @override
+         */
+        _render() {
+            const self = this;
+            this._super.apply(this, arguments);
+
+            if (this.showRecurringRevenue) {
+                const startMRR = this.prevTotalRecurringRevenue;
+                let endMRR = this.totalRecurringRevenue;
+
+                if (this.activeFilter.value) {
+                    endMRR = 0;
+                    this.columnState.data.forEach((record) => {
+                        if (this.activeFilter.value === record.data[this.fieldName] ||
+                            (this.activeFilter.value === '__false' && !record.data[this.fieldName])) {
+                            endMRR += parseFloat(record.data[this.recurringRevenueSumField]);
+                        }
+                    });
+                }
+
+                this.prevTotalRecurringRevenue = endMRR;
+
+                if (startMRR !== undefined && (endMRR > startMRR || this.activeFilter.value) && this.ANIMATE) {
+                    $({currentValue: startMRR}).animate({currentValue: endMRR}, {
+                        duration: 1000,
+                        start: function () {
+                            // always use 'o_kanban_grow' class due to limited length
+                            self.$counter.removeClass('o_kanban_grow_huge');
+                            self.$counter.addClass('o_kanban_grow');
+                            self.$mrrCounter.addClass('o_kanban_grow');
+                        },
+                        step: function () {
+                            self.$recurringRevenueNumber.html(utils.human_number(this.currentValue));
+                        },
+                        complete: function () {
+                            self.$recurringRevenueNumber.html(utils.human_number(this.currentValue));
+                            self.$counter.removeClass('o_kanban_grow');
+                            self.$mrrCounter.removeClass('o_kanban_grow');
+                        },
+                    });
+                } else {
+                    this.$recurringRevenueNumber.html(utils.human_number(endMRR));
+                }
+            }
+        },
+
+        /**
+         * @private
+         * @override
+         */
+        _notifyState() {
+            this._super.apply(this, arguments);
+            if (this.showRecurringRevenue) {
+                this.trigger_up('set_progress_bar_state', {
```

Comment:
Won't we have two triggers (one for base method, one for inherit) ? Shouldn't we instead try to have one trigger with all values ?